src/script/chem/molfile/molfile.js

```diff
@@ -386,6 +387,53 @@ Molfile.prototype.writeBondBlock3000 = function (molecule) {
 	this.writeCR('M  V30 END BOND');
 };
 
+/**
+ * @param {Struct} molecule
+ *
+ * Example collection block:
+ * 	M  V30 BEGIN COLLECTION
+ * 	M  V30 MDLV30/STEABS ATOMS=(2 3 4)
+ * 	M  V30 MDLV30/STEREL4 ATOMS=(1 2)
+ * 	M  V30 MDLV30/STERAC5 ATOMS=(2 1 6)
+ * 	M  V30 END COLLECTION
+ */
+Molfile.prototype.writeCollectionBlock3000 = function (molecule) {
+	const { abs, rac, rel } = molecule.enhancedStereo;
+	this.writeCR('M  V30 BEGIN COLLECTION');
+
+	// Absolute stereochemistry collection
+	// eg: M  V30 MDLV30/STEABS ATOMS=(2 3 4)
+	if (abs.length) {
+		const atomIndices = abs.map((id) => id + 1).join(' ');
+		let absoluteDetails = this.getMol3000Prefix();
+		absoluteDetails += `MDLV30/STEABS ATOMS=(${abs.length} ${atomIndices})`;
+		const absoluteDetailsLines = absoluteDetails.match(/.{1,70}/g);
```

Comment:
as offlined with @apar03 , we'll have a util do these text wrapping logic in the future